ios/engine/KMEI/KeymanEngine/Classes/Categories/UIView+Helpers.h

```diff
@@ -1,16 +0,0 @@
-//
-//  UIView+Helpers.h
-//  Keyman Engine
-//
-//  Created by Patrick Weekes on 2012-04-28.
-//  Copyright (c) 2017 SIL International. All rights reserved.
-//
-
-#import <UIKit/UIKit.h>
-
-@interface UIView (Helpers)
-
-- (NSString *)allSubviewsAsString;
-- (UIView *)findFirstResponder;
```

Comment:
Hmm, it seems that UIView+Helpers has disappeared now?  I assume this is because we're using WebViews exclusively now, or that our use of them replaced the areas that required these functions?

That being stated... I seem to recall `findFirstResponder` being referenced by [this block](https://github.com/keymanapp/keyman/pull/378#discussion-diff-147031706R1661) of code from my Manager.swift review.  (Follow the 'missing code' link.)

Reply:
`allSubviewsAsString` as unused and `findFirstResponder` was only used by that deleted block of code.